projects/ion/src/lib/smart-table/smart-table.component.spec.ts

```diff
@@ -661,6 +660,18 @@ describe('Table without Data', () => {
     await sut(tableWithoutData);
     expect(screen.getByTestId('ion-checkbox')).toBeDisabled();
   });
+
+  it('Should add a (-) when there is no data in the cell', async () => {
+    const customData = {
+      ...defaultProps,
+    };
+    customData.config.data = [{ name: '', height: 0, mass: 100 }];
+    await sut(customData);
+    const heightCell = screen.getByTestId('row-0-height');
+    const nameCell = screen.getByTestId('row-0-name');
+    expect(heightCell).toHaveTextContent('-');
+    expect(nameCell).toHaveTextContent('-');
```

Comment:
```suggestion
    expect(heightCell).toHaveTextContent(0);
```

for this case should be 0